# debug info for TA11 controller
std_msgs/Header header

float64[] k
float64[] delta_F
float64[] delta_p
float64[] cur_p
float64[] new_p
float64[] tra_p